test/clt-tests/buddy/test-fuzzy-search.rec

```diff
@@ -9,8 +9,9 @@ apt-get install jq -y > /dev/null; echo $?
 debconf: delaying package configuration, since apt-utils is not installed
 0
 ––– input –––
-php -d memory_limit=-1 ./test/clt-tests/scripts/load_us_names_min_infix_len.php 1000 10 1000000 5 2 > /dev/null
+php -d memory_limit=2G ./test/clt-tests/scripts/load_names_attr.php 100000 4 1000000 1 2G 1 2 > /dev/null; echo $?
```

Comment:
100K batch and 1M docs vs 1K batch and 5M docs - all on purpose?

Reply:
Fixed